app.js

```diff
@@ -13,6 +13,49 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function stateToUrl(state) {
+  var qsParams = Object.keys(state);
+
+  if (!qsParams.length) {
+    return "";
+  }
+
+  return "?" + qsParams.map(function(param) {
```

Comment:
I was thinking if I could use it, because the compatiblity section shows "No support" for IE and Android.

If you are okay with that, I will change it now.